packages/react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -209,7 +217,7 @@ export const FocusTrapZone: React.FunctionComponent<IFocusTrapZoneProps> & {
     if (ev.target !== ev.currentTarget && !(ev.target === firstBumper.current || ev.target === lastBumper.current)) {
       // every time focus changes within the trap zone, remember the focused element so that
       // it can be restored if focus leaves the pane and returns via keystroke (i.e. via a call to this.focus(true))
-      internalState.previouslyFocusedElementInTrapZone = ev.target as HTMLElement;
+      internalState.previouslyFocusedElementInTrapZone = getEventTarget(ev.nativeEvent) as HTMLElement;
```

Comment:
Do you need to use `nativeEvent` here?

Reply:
Yes, because we need to inspect the result for [`composedPath()`](https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath) which is not available on[ `SyntheticEvent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1242-L1258).